audeer/core/io.py

```diff
@@ -420,14 +428,15 @@ def list_file_names(
         list of path(s) to file(s)
 
     Raises:
-        NotADirectoryError: if ``os.path.dirname(path)``
-            is not a directory
-        FileNotFoundError: if ``os.path.dirname(path)``
-            does not exists
+        NotADirectoryError: if ``path`` is a directory
+            that does not exist,
```

Comment:
It will also raise the error if `path` is not a pattern, but a file that does not exist.
So maybe change to
```suggestion
        NotADirectoryError: if ``path`` does not exist,
```

or to 

```suggestion
        NotADirectoryError: if ``path`` is a directory or file
            that does not exist,
```